<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employeewindow</class>
 <widget class="QDialog" name="employeewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="subjectComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Przedmiot</string>
   </property>
  </widget>
  <widget class="QTableView" name="studentsTableView">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="gradeSpinBox">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>60</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ocena</string>
   </property>
  </widget>
  <widget class="QPushButton" name="assignGradeButton">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>120</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Zapisz ocene</string>
   </property>
  </widget>
  <widget class="QListWidget" name="attendanceListWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveAttendanceButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Zapisz frekwencjÄ™</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditAttendance">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="subject2ComboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="weekdayComboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>450</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="everyTwoWeeksCheckBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Co 2 tygodnie?</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveScheduleButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>540</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Zapisz harmonogram</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>304</width>
     <height>190</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>210</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="startTimeEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>570</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="endTimeEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>590</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
